.github/workflows/ci-md.yml

```
@@ -15,11 +15,16 @@ on:
 
 jobs:
   build-test:
-    runs-on: ubuntu-latest
-
     strategy:
       matrix:
-        version: [net6.0]
+        os: [ windows-latest, ubuntu-latest ]
+        version: [ net462, net6.0, net7.0 ]
+        exclude:
+        - os: ubuntu-latest
+          version: net462
+        - os: ubuntu-latest
+          version: net7.0
 
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ${{ matrix.os }}
```

should use the matrix here

---

Should I?
The old `.github/workflows/windows-ci-md.yml` used `ubuntu-latest`, It's just to run the `echo` command.

---

When you set the Required rules in github, it's going to be based on a string containing all the values in the matrix.

For example:
- Build, windows-latest, net462
- Build, ubuntu-latest, net6

The -md workflow will need to generate those same names to match the required rules.